x/group/keeper/msg_server_test.go

```diff
@@ -1806,6 +1808,15 @@ func (s *TestSuite) TestSubmitProposal() {
 				s.Require().Contains(fromBalances, sdk.NewInt64Coin("test", 9900))
 				toBalances := s.bankKeeper.GetAllBalances(sdkCtx, addr2)
 				s.Require().Contains(toBalances, sdk.NewInt64Coin("test", 100))
+				events := sdkCtx.EventManager().ABCIEvents()
```

Comment:
nit: maybe it could be simplified in a function that takes ABCIEvents, and returns a bool if an event is present?
This way this would limit code duplication in those tests.